lib/chartjs.dart

```diff
@@ -31,6 +31,7 @@ class Chart {
   external factory Chart(
       dynamic /*String|CanvasRenderingContext2D|CanvasElement|ArrayLike<CanvasRenderingContext2D|CanvasElement>*/ context,
       ChartConfiguration options);
+  external get scales;
```

Comment:
Does this have a type?

Reply:
it doesn't work if I put a type because it must be accessed like a map ['prop']. Asked on gitter and SO how to solve but didn't find a solution